pkg/asset/handler/get.go

```diff
@@ -108,7 +122,7 @@ func (h *GetHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 
 		valid := imageprocessing.IsApplicableToHTTPResponse(resp)
-		if !valid || !hasPipeline {
+		if isHead || !valid || !hasPipeline {
```

Comment:
If it's `HEAD`, we should try not sending the body to client.

Reply:
I tried to do this but a weird error got printed to console saying that somebody wrote to body after header is flushed.

Reply:
I think replacing `resp.Body` with `ioutil.NopCloser(bytes.NewReader([]byte{}))` should work?